src/catfish-tools.h

```diff
@@ -0,0 +1,7 @@
+#ifndef ASTEROIDOS_CATFISH_TOOLS_H
+#define ASTEROIDOS_CATFISH_TOOLS_H
+
+namespace AsteroidOS::LCD_Tools::Catfish {
+	void SyncTime(int);
```

Comment:
None of the `SyncTime` methods actually use the `int` type. What's the use case for this? Maybe remove it?

Reply:
Nvm, this has something to do with the CLI11 callback signatures.